tests/kafkatest/tests/client/consumer_test.py

```diff
@@ -608,7 +617,7 @@ def test_valid_assignment(self, assignment_strategy=None, metadata_quorum=quorum
             consumer.start_node(node)
             self.await_members(consumer, num_started)
             wait_until(lambda: self.valid_assignment(self.TOPIC, self.NUM_PARTITIONS, consumer.current_assignment()),
-                timeout_sec=15,
+                timeout_sec=self.session_timeout_sec*2,
```

Comment:
Extend the timeout a bit as it can timeout for normal reconciliation.

Reply:
What the reasoning for using `self.session_timeout_sec*2`? Would using 30s also work?

Reply:
30s also works. Using `self.session_timeout_sec*2` is a rough assumption that it takes two heartbeats for a consumer to stabilize. In this method it will be 20s.